notebooks/05_operationalize/pipeline-demo/aml-pipeline-use-datebricks-as-compute-target.ipynb

```diff
@@ -0,0 +1,697 @@
+{
```

Comment:
<p>the databricks.egg_libraries path is specific to your workspace, when using the databricks_install.py script, I believe we write the egg to dbfs:/FileStore/jars/Recommenders.egg</p><p><br></p><p>we should provide instructions to use the databricks install script and change this to remove the hashed prefix </p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/643/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>